src/js/yeldaWebchatInjector.js

```diff
@@ -196,6 +196,9 @@ class YeldaChat {
     url = this.updateQueryStringParameter(url, 'assistantSlug', data.assistantSlug)
     url = this.updateQueryStringParameter(url, 'locale', data.locale)
 
+    if (data.platformSimulated) {
+      url = this.updateQueryStringParameter(url, 'platformSimulated', data.platformSimulated)
+    }
     if (data.hasOwnProperty('canBeClosed')) {
       url = this.updateQueryStringParameter(url, 'canBeClosed', data.canBeClosed)
     }
```

Comment:
please add a test 
it should not be very useful, but could be great to keep the habit
![image](https://user-images.githubusercontent.com/5340081/97306307-9920fb00-185e-11eb-8748-97bc84940c5f.png)